tests/Microsoft.Identity.Web.Test/TestTokenAcquisitionHost.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Xunit;
+
+namespace Microsoft.Identity.Web.Test
+{
+    public class TestTokenAcquisitionHost
+    {
+        [Fact]
+        public void TestTokenAcquisitionHostNotAspNetCore()
+        {
+            // When not adding Services.AddAuthentication, the host should be 
+            TokenAcquirerFactory tokenAcquirerFactory = TokenAcquirerFactory.GetDefaultInstance();
+            var host = tokenAcquirerFactory.Services.First(s => s.ServiceType.Name == "Microsoft.Identity.Web.ITokenAcquisitionHost");
```

Comment:
```suggestion
            var host = tokenAcquirerFactory.Services.First(s => s.ServiceType.Name == "ITokenAcquisitionHost");
```

Reply:
this test fails on .net6, I have tested that this change fixes it

Reply:
yes. it needs to use Fullname. my bad

Reply:
Your fix works too, @westin-m, but there could be ITokenAcquisitionHost in several namespaces (purely hypothetical, though)